app/assets/javascripts/qsets.js

```diff
@@ -1,9 +1,4 @@
 function initQsets() {
-  $('#qsets').on('change', function(event) {
-    var $selectedOption = $(this.options[this.selectedIndex]);
-    window.open($selectedOption.data('qset-url'), '_self');
-  });
-
```

Comment:
What was this function doing before? Was it removed because it addressed this specific issue or is it just generally no longer relevant?